pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/export/json/structured/QuestionRecord.java

```diff
@@ -38,12 +38,14 @@ public JsonElement serialize(QuestionRecord record, Type typeOfSrc, JsonSerializ
             } else if (record instanceof PicklistQuestionRecord) {
                 questionAnswer = ((PicklistQuestionRecord) record).getSelected();
             } else if (record instanceof CompositeQuestionRecord) {
-                questionAnswer = ((CompositeQuestionRecord) record).getAnswer();
+                //questionAnswer = ((CompositeQuestionRecord) record).getAnswer();
```

Comment:
Temporarily disable exporting composite in additional new format until mapping issue is fixed.
Additional new format is not yet used by DSM.